scripts/healthcheck.sh

```diff
@@ -54,5 +55,17 @@ if [[ ${TRANSMISSION} -ne 1 ]]; then
 	exit 1
 fi
 
+if [[ ${WEBPROXY_ENABLED} =~ YyTruetrue ]]; then
```

Comment:
I don't think the regex here is correct. It should be something like `"Y|y|True|true"`.